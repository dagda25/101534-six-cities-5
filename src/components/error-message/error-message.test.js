import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message";

const text = `test`;
const onClick = () => {};
describe(`Render ErrorMessage`, () => {
  it(`Render ErrorMessage`, () => {
    const tree = renderer
      .create(
          <ErrorMessage text={text} onClick={onClick}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
