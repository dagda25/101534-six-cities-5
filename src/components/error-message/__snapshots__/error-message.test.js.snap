// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ErrorMessage Render ErrorMessage 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "red",
      "borderRadius": "5px",
      "color": "white",
      "marginTop": "5px",
      "padding": "10px",
    }
  }
>
  test
</div>
`;
